### PR TITLE
get apk url and set apk url environment

### PR DESCRIPTION
**Pull request by oreno4649**
## Summary

I extracted the URL from the firebase output and set it to an environment variable

The following values are output to the environment variables
FIREBASE_CONSOLE_URL : The firebase console URL of the deployed app
FIREBASE_APP_DISTRIBUTION_URL : Share this release with testers who have access

## Supplement

Output may be incorrect if the firebase output format is changed.

## Issue

https://github.com/guness/bitrise-step-firebase-app-distribution/issues/65

### CODE REVIEW

File: step.sh [hunk 1]
```diff
@@ -240,7 +240,18 @@ fi
...
 
-eval "${submit_cmd}"
+output=$(eval $submit_cmd)
+echo $output
+
+# Adjust the number of `sed -n 2p` if the position of the URL changes in the output
+FIREBASE_CONSOLE_URL=$(echo $output | grep -Eo "(http|https)://[a-zA-Z0-9./?=-_%:-]*" | sed -n 2p)
+echo_info "firebase console url: ${FIREBASE_CONSOLE_URL}"
+envman add --key FIREBASE_CONSOLE_URL --value "${FIREBASE_CONSOLE_URL}"
+
```
Comment (guness) [inline step.sh:250]: is there any format of console URLs and distribution URLs?
We should not just rely on the order.

Comment (guness) [reply step.sh:250]: @oreno4649 ^^

Comment (faruk0344) [reply step.sh:250]: @guness 
As @gremech suggested extract the `release_id_short` is also good enough and put is to env variable.
Is it possible to apply this and release a new version. 

Actually I am waiting for it to integrate to my CI/CD for real project

**Comment by guness** [comment]
please target dev branch. I would like to use your commits.

**Comment by oreno4649** [comment]
@guness 
Sorry to keep you waiting. 😭 
I've changed the target and resolved the conflicts.

**Review by guness — APPROVED** [review]

**Comment by guness** [comment]
this is looking good. let's merge it. I might edit on the branch.

**Pull request merged**